apps/bfd-model/bfd-model-rif/src/main/resources/db/migration/V90__new_schema_rename_to_original.sql

```diff
@@ -0,0 +1,534 @@
+alter index ${logic.psql-only} if exists 
+    public.beneficiaries_new_ptd_cntrct_number_apr_id_idx rename to beneficiaries_ptd_cntrct_number_apr_id_idx;
+alter index ${logic.psql-only} if exists 
+    public.beneficiaries_new_ptd_cntrct_number_may_id_idx rename to beneficiaries_ptd_cntrct_number_may_id_idx;
+alter index ${logic.psql-only} if exists 
+    public.beneficiaries_new_ptd_cntrct_number_jun_id_idx rename to beneficiaries_ptd_cntrct_number_jun_id_idx;
+alter index ${logic.psql-only} if exists 
+    public.beneficiaries_new_ptd_cntrct_number_jul_id_idx rename to beneficiaries_ptd_cntrct_number_jul_id_idx;
+alter index ${logic.psql-only} if exists 
+    public.beneficiaries_new_ptd_cntrct_number_aug_id_idx rename to beneficiaries_ptd_cntrct_number_aug_id_idx;
+alter index ${logic.psql-only} if exists 
+    public.beneficiaries_new_ptd_cntrct_number_sept_id_idx rename to beneficiaries_ptd_cntrct_number_sept_id_idx;
+alter index ${logic.psql-only} if exists 
+    public.beneficiaries_new_ptd_cntrct_number_oct_id_idx rename to beneficiaries_ptd_cntrct_number_oct_id_idx;
+alter index ${logic.psql-only} if exists 
+    public.beneficiaries_new_ptd_cntrct_number_nov_id_idx rename to beneficiaries_ptd_cntrct_number_nov_id_idx;
+alter index ${logic.psql-only} if exists 
+    public.beneficiaries_new_ptd_cntrct_number_dec_id_idx rename to beneficiaries_ptd_cntrct_number_dec_id_idx;
+
+-- =====================
+-- beneficiaries_history
+-- =====================
+-- psql allows you to rename primary key
+${logic.psql-only} alter index if exists public.beneficiaries_history_new_pkey
+${logic.psql-only}    rename to beneficiaries_history_pkey;
+
+-- hsql doesn't support rename on primary key; we'll drop and re-create primary key
+${logic.hsql-only} alter table public.beneficiaries_history_new
+${logic.hsql-only}     drop constraint public.beneficiaries_history_new_pkey;
+${logic.hsql-only} alter table public.beneficiaries_history_new
+${logic.hsql-only}     add constraint beneficiaries_history_pkey primary key (bene_history_id);
+
+-- change the name of the index for bene_id
+alter index ${logic.psql-only} if exists 
+    public.beneficiaries_history_new_bene_id_idx rename to beneficiaries_history_bene_id_idx;
+  
+-- change the name of the index for bene_crnt_hic_num
+alter index ${logic.psql-only} if exists 
+    public.beneficiaries_history_new_hicn_idx rename to beneficiaries_history_hicn_idx;
+    
+-- change the name of the index for mbi_hash
+alter index ${logic.psql-only} if exists 
+    public.beneficiaries_history_new_mbi_hash_idx rename to beneficiaries_history_mbi_hash_idx;
+
+-- for hsql, we previously dropped the primary key with cascade, so we need
+-- to actually create the fk constraint
+alter table if exists public.beneficiaries_history_new
+${logic.hsql-only} add constraint beneficiaries_history_to_beneficiaries foreign key (bene_id)
+${logic.hsql-only}     references public.beneficiaries_new (bene_id);
```

Comment:
will this reference be updated when we update the table name further down this script, or does this need to get updated again after the rename? (Or does this ref not matter after this line?)

Reply:
apparently my response didn't take. Yes, that is correct. During the table rename(s), any constraint will be correctly set to the new talbe name.